src/main/java/reactor/netty/resources/PooledConnectionProvider.java

```diff
@@ -80,13 +80,15 @@
 	final String                          name;
 	final Map<SocketAddress, PoolFactory> poolFactoryPerRemoteHost = new HashMap<>();
 	final PoolFactory                     defaultPoolFactory;
+	MeterRegistrar 				  		  registrar;
```

Comment:
`registrar` is not a configuration on ConnectionProvider level, it is related to a particular connection pool.
Let's take this example

```
ConnectionProvider.builder("test")
                  .maxConnections(2)
                  .metrics(true, () -> new CustomImpl1())
                  .forRemoteHost(InetSocketAddress.createUnresolved("localhost", server.port()),
                                 spec -> spec.maxConnections(1).metrics(true, () -> new CustomImpl2()))
                  .build();
```